apis/istio/v1alpha3/virtualservice_types.go

```diff
@@ -474,7 +476,48 @@ type TCPRoute struct {
 	// Currently, only one destination is allowed for TCP services. When TCP
```

Comment:
Is this comment still up to date? If so, what are the semantics now with this being an array?

Reply:
I think multiple destinations are allowed, I'll remove the comment.